src/game/characters/PlayerCharacter.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseCharacter, BaseStats } from "./BaseCharacter";
+
+export class PlayerCharacter extends BaseCharacter {
+    private name: string;
+    private level: number;
+    sprite: Phaser.Types.Physics.Arcade.SpriteWithDynamicBody;
+
+    constructor(name: string, level: number, stats: BaseStats, sprite: Phaser.Types.Physics.Arcade.SpriteWithDynamicBody) {
+        super(stats);
+        this.name = name;
+        this.level = level;
+        this.sprite = sprite;
+    }
+
+    createAnimations(scene: Phaser.Scene, asepriteKey: string): void {
+        scene.anims.createFromAseprite(asepriteKey);
+    }
+
+    focusCamera(scene: Phaser.Scene): void {
+        scene.cameras.main.startFollow(this.sprite, true, 0.1, 0.1);
+    }
+    
+    getName(): string {
+        return this.name;
+    }
+
+    getLevel(): number {
+        return this.level;
+    }
```

Comment:
I don't really think there's a need for this, we can just do `player.level` or `player.name`  no ? Any reason why we should have this method ? :thinking: